xarray/core/dataset.py

```diff
@@ -1726,19 +1726,58 @@ def data_vars(self) -> DataVariables:
     def set_coords(self: T_Dataset, names: Hashable | Iterable[Hashable]) -> T_Dataset:
         """Given names of one or more variables, set them as coordinates
 
-        Parameters
-        ----------
-        names : hashable or iterable of hashable
-            Name(s) of variables in this dataset to convert into coordinates.
+                Parameters
```

Comment:
Accidental indentation again here, need to change that back.

Reply:
pre-commit run is actually changing the indentation, even after it's correctly indented.